plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/BaseKuduConnectorTest.java

```diff
@@ -24,9 +24,10 @@
 import static io.trino.spi.type.VarcharType.VARCHAR;
 import static io.trino.testing.MaterializedResult.resultBuilder;
 import static io.trino.testing.assertions.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
 
-public abstract class BaseKuduDistributedQueries
-        extends AbstractTestDistributedQueries
+public abstract class BaseKuduConnectorTest
+        extends BaseConnectorTest
```

Comment:
If you are using `BaseConnectorTest` then you should also remove `AbstractKuduIntegrationSmokeTest` as smoke tests are already in `BaseConnectorTest`.

Reply:
@kokosing Took a look at removing this. I am a bit hesitant because each of the integration smoke tests implementations have a separate schema inference -- disabled, empty, or standard -- which is given to as an input parameter to a QueryRunner. This seems intentional to me, but I am not familiar with the Kudu connector tests. Just trying to add support for testing the latest. Let me know your thoughts.

I've rebased this PR to the latest master.

Reply:
> each of the integration smoke tests implementations have a separate schema inference -- disabled, empty, or standard 
So you can have a dedicated instance of `BaseKuduConnectorTest` for each case. Or you can create a `BaseKuduConnectorSmokeTest` (that extends `BaseConnectorSmokeTest`) and use it for each case.